tests/integrations/python/tests/test_openai.py

```
@@ -2117,30 +2117,133 @@ def test_37_responses_streaming_with_tools(self, test_config, provider, model, v
+
+            # Turn 2: Follow-up that requires recalling previous reasoning
+            # Build context with previous response output (includes reasoning items)
+            second_turn_input = list(response1.output) + [
+                {
+                    "role": "user",
+                    "content": (
+                        "Now, what if the first train had left 30 minutes earlier (at 1:30 PM) instead? "
+                        "How would that change when they meet? Use your previous reasoning as a foundation."
+                    ),
+                }
+            ]
```

Ruff (0.15.6)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)